automation/README.md

```diff
@@ -0,0 +1,77 @@
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository uses the `check-patch` event at the moment,
+to verify incoming patches.
+
+Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
+triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
+which executes the [`automation/check-patch.sh`](./check-patch.sh) script.
+
+This script will execute the Ansible playbook [`./control.yml`](../control.yml)
+which wraps entire testing flow.
+
+
+## `control.yml` Ansible playbook
+
+This Ansible playbook is wrapper for entire testing flow which is composed from
+following steps (partial playbooks)
+* [provision testing environment](#testing-environment)
+* [deploy OpenShift](#deploy-openshift)
+* [install KubeVirt](#install-kubevirt)
+
+This playbook is executed inside of CentOS 7.4 mock.
+There is additional software installed, please read
+[automation/check-patch.packages](./check-patch.packages) for complete list.
+
+Parameters and usage of this playbook is described at
+[Deploy new Kubernetes or OpenShift cluster and KubeVirt with Lago](../README.md#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago).
+
+This is list of playbooks which are executed.
+* [control.yml](../control.yml)
+  * [deploy-with-lago.yml](../deploy-with-lago.yml)
+  * [deploy-openshift.yml](../deploy-openshift.yml)
+  * [install-kubevirt-release.yml](../install-kubevirt-release.yml)
+
+### Testing environment
+
+To provision testing environment the [`./deploy-with-lago.yml`](../deploy-with-lago.yml) playbook is used.
+
+Testing environment is populated by
+[Lago](https://github.com/lago-project/lago) project, it provisions desired
+resources to match requirements described in Lago configuration file.
+This configuration file is located in root of repository
+[`LagoInitFile.yml`](../LagoInitFile.yml).
+
+It will provision three virtual machines with CentOS 7.4
+* 1x master node
+* 2x compute node
+
+
+| Node type | Memory | CPUs | Disks | Ansible Groups |
+| ---- | ---- | ---- | ---- | ---- |
+| master | 4Gb | ? | * root<br> * docker\_storage (10Gb)<br> * docker\_lib (10Gb)<br> * main\_nfs (101Gb) | * masters<br> * nodes<br> * nfs<br> * etcd |
+| node   | 2Gb | ? | * root<br> * docker\_storage (10Gb)<br> * docker\_lib (10Gb) | * nodes |
+
+
+### Deploy OpenShift
+
+To deploy OpenShift the [`./deploy-openshift.yml`](../deploy-openshift.yml) playbook is used.
+
+Test flow deploys OpenShift in two versions:
+
+* OpenShift 3.7
+* OpenShift 3.9
+
+### Install KubeVirt
+
+To install KubeVirt on top of  OpenShift the [`./install-kubevirt-release.yml`](../install-kubevirt-release.yml) playbook is used.
+
+Test flow installs KubeVirt v0.2.0 .
+
+[ovirt-ci-system-doc]: http://ovirt-infra-docs.readthedocs.io/en/latest/CI/Build_and_test_standards/index.html
```

Comment:
Maybe it will help to add here a link to 'contributing' and say the project is open to integrating additional 'kubevirt projects' or integrations which can also run on each PR once submitted?

Reply:
Addressed